src/data/resolvers/motions.ts

```diff
@@ -928,6 +928,44 @@ export const motionsResolvers = ({
         bigNumberify(fundamentalChainId),
       );
     },
+    async timeoutPeriods({
```

Comment:
Why did you add this logic in again ?

We already have it in the `motionTimeoutPeriods` resolver.

If you need to use it in a different resolver as well, just extract it somewhere you can use in both places, please don't duplicate code.

Reply:
I did it for testing and I was meaning to extract it but then forgot

Reply:
I've done it now and you can rereview.